manage-mysql-user.py

```diff
@@ -424,12 +433,25 @@ def handler(event, context):
         mysql_master_password_source,
         mysql_master_password_source_type,
     )
-    execute_statement(
-        "GRANT ALL ON `{}`.* to '{}'@'%';".format(database, mysql_user_username),
-        mysql_master_username,
-        mysql_master_password_source,
-        mysql_master_password_source_type,
-    )
+
+    if "privileges" in event.keys():
+        if len(event["privileges"]) > 0:
+            privileges = event["privileges"]
+            logger.info(
+                f"Granting {privileges} privileges to MySQL user {mysql_user_username}"
+            )
+            execute_statement(
+                "GRANT {} ON `{}`.* to '{}'@'%';".format(
```

Comment:
pretty sure that `if event["privileges"] not in ["ALL", "SELECT", ""]` on line 322 prevents a SQL injection here

Reply:
well yes, that's the logic behind it